<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:#FFFFFF
;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditQuatDiscos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantDiscos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Quantidade de Discos &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTorreDeHanoi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;TORRE DE HANÃ“I&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_QuantJogadas">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Quantidade de Jogadas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuatJogadas">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>A --&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAC">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>520</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>A --&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBA">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>B --&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>C --&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCA">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>C --&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBC">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#0066ff;</string>
    </property>
    <property name="text">
     <string>B --&gt; C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="base">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFFFFF;
border-radius:5px;
background-color:#964b00;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIniciar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:#00ff00;</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Users/pedro/Downloads/botao-start.png</normaloff>../../Users/pedro/Downloads/botao-start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTorreA">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>40</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFFFFF;
border-radius:5px;
background-color:#964b00;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTorreB">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>40</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#964b00;
color:#FFFFFF;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTorreC">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>40</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:#964b00;
color:#FFFFFF;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Users/pedro/Downloads/icon-information-png-7.png</normaloff>../../Users/pedro/Downloads/icon-information-png-7.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>lineEditQuatDiscos</zorder>
   <zorder>label_QuantDiscos</zorder>
   <zorder>LabelTorreDeHanoi</zorder>
   <zorder>label_QuantJogadas</zorder>
   <zorder>lineEditQuatJogadas</zorder>
   <zorder>pushButtonAB</zorder>
   <zorder>pushButtonAC</zorder>
   <zorder>pushButtonBA</zorder>
   <zorder>pushButtonCB</zorder>
   <zorder>pushButtonCA</zorder>
   <zorder>pushButtonBC</zorder>
   <zorder>pushButtonIniciar</zorder>
   <zorder>textEditTorreA</zorder>
   <zorder>textEditTorreB</zorder>
   <zorder>textEditTorreC</zorder>
   <zorder>base</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
